.github/workflows/kustomize-validation.yml

```diff
@@ -0,0 +1,47 @@
+name: Kustomize Validation
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  kustomize-validation:
+    name: Validate Kustomization Files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Kustomize
```

Comment:
do we need kustomize? It should be preinstalled in the github runners images: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md